README.md

```diff
@@ -263,7 +277,7 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
             "threads": 3,
 
             // The maximum size allowed for the body of a request.
-            "max_rbuffer_size_in_bytes": 8388608,
+            "max_size_of_body_in_bytes": 8388608,
```

Comment:
Should this be `max_size_of_request_body_in_bytes`?

Reply:
Oh, I see. I personally find that confusing, but I won't push for it.

Reply:
yeah, kind of agree - but let's let it ride.

Reply:
Sounds like it needs to be changed. I think I'm starting to agree.

Reply:
Changed to suggested name.